integration/experiment/common_args.py

```diff
@@ -86,3 +86,9 @@ def add_iterations(parser):
     parser.add_argument('--iterations', dest='iterations',
                         action='store', type=int, default=2,
                         help='number of iterations to launch')
+
+
+def add_use_stdev(parser):
```

Comment:
Maybe would be nicer to select the error bar type?

--error-bar=stdev

or

--error-bar=minmax

I think it depends on how many options we expect to have the error bar choice.

Reply:
maybe we can change this when we add a third option?  otherwise, it looks like we can use 'choices' parameter from argparse to validate this option

Reply:
also, would the default then be no error bars, or default to minmax?

Reply:
My vote is for minmax, but I'm fine with this coming in later.

Reply:
I'll work on a separate PR to change the option to string and refactor the repeated code for error bars